.github/workflows/x-keycloak-quickstarts.yml

```
@@ -32,12 +41,22 @@ jobs:
           token: ${{ secrets.GH_TOKEN }}
           ref: ${{ inputs.target-branch }}
 
-      - name: Set server version
+      - name: Set version for server or js
```

```suggestion
      - name: Set version for server or js or client
```
Or perhaps something more generic?